Makefile

```
@@ -70,7 +70,19 @@ xcode: ## Generate and open Xcode workspace
 	open "Just Speak to It.xcworkspace"
 
 .PHONY: archive
-archive: ## Create Xcode archive for App Store
+archive: ## Create Xcode archive of the current workspace (direct/Developer ID flavour)
```

checkmake (0.3.2)

[warning] Target body for "archive" exceeds allowed length of 5 lines (7).

(maxbodylength)